via/views/api/youtube.py

```diff
@@ -0,0 +1,1328 @@
+from pyramid.httpexceptions import HTTPNotFound
+from pyramid.view import view_config
+
+
+@view_config(route_name="api.youtube.transcript", permission="api", renderer="json")
+def transcript(request):
+    """Return the transcript of a given YouTube video."""
+
+    transcript_id = request.matchdict["transcript_id"]
+
+    if transcript_id != "1":
+        raise HTTPNotFound()
+
+    # pylint:disable=too-many-lines
+    return {
+        "data": {
+            "type": "transcripts",
+            "id": transcript_id,
+            "attributes": {
```

Comment:
The `"data"`, `"type"`, `"id"` and `"attributes"` are [JSON:API](https://jsonapi.org/)-style design. See https://github.com/hypothesis/via/issues/990#issuecomment-1589700767